Packs/SymantecDLP/Integrations/SymantecDLPV2/SymantecDLPV2.py

```diff
@@ -560,21 +695,33 @@ def list_incidents_command(client: Client, args: Dict[str, Any]) -> CommandResul
     limit = arg_to_number(args.get('limit', 50))
     page = arg_to_number(args.get('page', 1))
     page_size = arg_to_number(args.get('page_size'))
-
-    incidents_result = client.get_incidents_request(creation_date, status_ids, severities_dlp, incident_types_dlp,
-                                                    limit * page)  # type: ignore
+    raw_filter = args.get('raw_filter')
+    try:
+        incidents_result = client.get_incidents_request(creation_date, status_ids, severities_dlp, incident_types_dlp,
+                                                        limit * page, None, raw_filter)  # type: ignore
+    except DemistoException as e:
+        if '400' in str(e):
+            return_error = 'Error 400: Bad filter. Ensure the filter meets the requirements shown at https://apidocs.securitycloud.symantec.com/#/doc?id=incidentlists.'
+        else:
+            raise e
     incidents_result = get_incidents_of_current_page(limit, page, page_size,
                                                      incidents_list=incidents_result['incidents'])
-    list_incidents_hr = get_readable_output_incidents_list(incidents_result)
-    context_incidents_list = get_context_incidents_list(incidents_result)
+    if raw_filter:
+        list_incidents_hr = incidents_result
+        context_incidents_list = incidents_result
+        output_headers = list(incidents_result[0].keys())
+    else:
+        list_incidents_hr = get_readable_output_incidents_list(incidents_result)
+        context_incidents_list = get_context_incidents_list(incidents_result)
+        output_headers = ['ID', 'Severity', 'Status', 'Creation Date', 'Incident Type', 'Message Type', 'Policy ID',
+                          'Match Count']
```

Comment:
```suggestion
  list_incidents_hr = get_readable_output_incidents_list(incidents_result)
  context_incidents_list = get_context_incidents_list(incidents_result)
```


why does the context / human readable needs to be different when `raw_filter` was provided? it should be the same output as before.

Reply:
The raw filter also includes the columns to select from the SDLP console. The default method of returning the incident list only works for a static set of columns and you get unpredictable behavior if the columns requested by the raw filter do not exactly match the columns mapped in the get_readable_output_incidents_list and get_context_incidents_list functions.